src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -151,16 +151,16 @@ key *docker-compose-options-split-regex*. If you supply e.g. "!SPLIT!" as value,
 
 === Docker_File task
 
-The basic behavior of a Docker_File task is to create an image with Docker build and to create a container based on the built image. At the end of the job, the created image and container are deleted. Advanced options will allow to parametrize these actions.
+The basic behavior of a Docker_File task is to build an image and run a container using the built image. At the end of the task (the dockerfile execution), the container is stooped and the image is deleted. 
```

Comment:
Reformulation:
The main behavior of a Docker_File task is to first build an image and then run a container instance from it. Once the execution is done, the container is stooped and the built image is deleted.